packages/connect-web-test/src/fetch.spec.ts

```diff
@@ -107,5 +107,34 @@ describe("custom fetch", function () {
       expect(response.json).toHaveBeenCalledTimes(1); // eslint-disable-line @typescript-eslint/unbound-method
       expect(response.arrayBuffer).toHaveBeenCalledTimes(0); // eslint-disable-line @typescript-eslint/unbound-method
     });
+    it("should support polyfills", async function () {
```

Comment:
I wondered for a second what this means. The point is that we don't want to resolve a reference to `globalThis.fetch` _before_ a patch or polyfill has been applied, right?

This makes sense, good call 👍 

Can we find a more descriptive PR title and test name? And can we cover gRPC-web (at least unary) as well?